### PR TITLE
Removing local job as we should always use LSF

### PR DESCRIPTION
**Pull request by thomasmaurel**

### CODE REVIEW

File: perl/Bio/EnsEMBL/Healthcheck/Pipeline/RunStandaloneHealthchecksParallel_conf.pm [hunk 1]
```diff
@@ -13,8 +13,7 @@ sub pipeline_analyses {
   return [ {
       -logic_name => 'RunStandaloneHealthcheckFactory',
       -module =>
-        'Bio::EnsEMBL::Healthcheck::Pipeline::RunStandaloneHealthcheckFactory',
-      -meadow_type => 'LOCAL',
+        'Bio::EnsEMBL::Healthcheck::Pipeline::RunStandaloneHealthcheckFactory'
```
Comment (marcoooo) [inline perl/Bio/EnsEMBL/Healthcheck/Pipeline/RunStandaloneHealthchecksParallel_conf.pm]: This line is missing the trailing ','

File: perl/Bio/EnsEMBL/Healthcheck/Pipeline/RunStandaloneHealthchecksParallel_conf.pm [hunk 2]
```diff
@@ -37,8 +36,7 @@ sub pipeline_analyses {
       -flow_into => { 2 => ['?accu_name=hc_output&accu_address=[]'] } }, {
       -logic_name => 'RunStandaloneHealthcheckMerge',
       -module =>
-        'Bio::EnsEMBL::Healthcheck::Pipeline::RunStandaloneHealthcheckMerge',
-      -meadow_type => 'LOCAL',
+        'Bio::EnsEMBL::Healthcheck::Pipeline::RunStandaloneHealthcheckMerge'
```
Comment (marcoooo) [inline perl/Bio/EnsEMBL/Healthcheck/Pipeline/RunStandaloneHealthchecksParallel_conf.pm]: This line is missing the trailing ','

**Comment by thomasmaurel** [comment]
Thanks for spotting this @marcoooo. This was obviously made on purpose to test you ;-)

**Comment by marcoooo** [comment]
> Thanks for spotting this @marcoooo. This was obviously made on purpose to test you ;-)

And this time I read every lines :-) So I guess I passed the test !

**Comment by thomasmaurel** [comment]
@marcoooo: Test passed with honours :-)

**Pull request merged**